utils/data.py

```diff
@@ -89,146 +89,7 @@ def str():
                [-1,0],
                [3,0,0,-1,3,3],
                [5,0,1,0,3,-1,5,6,5,8],
-               oj.name2idx(oj.parent_joint_names)]  #full skeleton possesses openpose topology
-    parents_list = list(map(np.array, parents_list))  # convert list items to np
```

Comment:
no need for class joint's parents_list any more?

Reply:
Its the same variable, just a conversion to numpy array which we don't need anymore.